src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_user_profile.scss

```diff
@@ -58,6 +58,64 @@
     }
 }
 
+.user-profile-image-edit {
+    $image-base-size: 200px;
+    
+    @include rem(height, $image-base-size);
+    @include rem(width, $image-base-size);
+    position: relative;
+
+    // position nicely on top of the username
+    margin-top: 60px;
+    margin-bottom: -60px;
+
+    // center
+    margin-right: auto;
+    margin-left: auto;
+
+    .user-profile-image {
+        @include rem(height, $image-base-size);
+        @include rem(width, $image-base-size);
+    }
+    
+    &.user-profile-image-edit-uploading {
+        @include rem(width, $image-base-size);
+        height: auto;
+    }
+
+    .user-profile-image-edit-button {
+        @include rem(width, $image-base-size);
+        @include rem(height, $image-base-size);
+        @include border-radius(50%);
+
+        position: absolute;
+        top: 0;
+        left: 0;
+        background-color: rgba(0, 0, 0, 0.5);
+
+        visibility: hidden;
+
+        > span {
+            // vertically centered
+            @include rem(height, $image-base-size);
+            @include rem(width, $image-base-size);
+            display: table-cell;
+            vertical-align: middle;
+            // We actually want the 'button' to wrap
+            white-space: normal;
+        }
+    }
+
+    &:hover .user-profile-image-edit-button {
+        visibility: visible;
+    }
+
+    .user-profile-image-edit-button.user-profile-image-edit-button-force-visible {
+        @extend .button:hover;
+        visibility: visible;
+    }
+}
+
 .user-activity-overview {
     @include box-sizing(border-box);
```

Comment:
It seems this default is in master already? I'll remove this and rebase the pull request then.